src/net.h

```diff
@@ -729,7 +729,7 @@ class CNode
 
     // flood relay
     std::vector<CAddress> vAddrToSend;
-    CRollingBloomFilter addrKnown;
+    std::unique_ptr<CRollingBloomFilter> addrKnown;
```

Comment:
```suggestion
    std::unique_ptr<CRollingBloomFilter> m_addr_known;
```

style-nit: Should use new style for new members